### PR TITLE
Add logback configuration doc

### PR DESCRIPTION
**Pull request by trask**
See #7389

### CODE REVIEW

**Review by laurit — APPROVED** [review]

File: instrumentation/logback/logback-appender-1.0/javaagent/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Settings for the Logback Appender instrumentation
+
+| System property | Type    | Default | Description                                          |
+|---|---------|--|------------------------------------------------------|
+| `otel.instrumentation.logback-appender.experimental-log-attributes` | Boolean | `false` | Enable the capture of experimental span attributes `thread.name` and `thread.id`. |
+| `otel.instrumentation.logback-appender.experimental.capture-code-attributes` | Boolean | `false` | Enable the capture of [source code attributes]. (note: capturing source code attributes at logging sites can add a performance overhead) |
```
Comment (theletterf) [inline instrumentation/logback/logback-appender-1.0/javaagent/README.md:6]:
```suggestion
| `otel.instrumentation.logback-appender.experimental.capture-code-attributes` | Boolean | `false` | Enable the capture of [source code attributes]. Note that capturing source code attributes at logging sites might add a performance overhead. |
```

File: instrumentation/logback/logback-appender-1.0/javaagent/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+| `otel.instrumentation.logback-appender.experimental.capture-marker-attribute` | Boolean | `false` | Enable the capture of Logback markers as attributes. |
+| `otel.instrumentation.logback-appender.experimental.capture-mdc-attributes` | String  |  | List of MDC attributes to capture (`*` is a special value to capture them all). |
```
Comment (theletterf) [inline instrumentation/logback/logback-appender-1.0/javaagent/README.md:8]:
```suggestion
| `otel.instrumentation.logback-appender.experimental.capture-mdc-attributes` | String  |  | List of MDC attributes to capture. Use the wildcard character `*` to capture all attributes. |
```

**Review by theletterf — APPROVED** [review]

**Comment by jack-berg** [comment]
I believe this applies to the library version of this as well. Separately, Log4j and JUL have the same undocumented features. (JUL only has support for `otel.instrumentation.java-util-logging.experimental-log-attributes`)

**Review by mateuszrzeszutek — APPROVED** [review]

**Comment by trask** [comment]
> I believe this applies to the library version of this as well. Separately, Log4j and JUL have the same undocumented features. (JUL only has support for `otel.instrumentation.java-util-logging.experimental-log-attributes`)

I've sent doc for the Log4j and JUL javaagent settings, and I've updated #6947 to mention the missing library documentation

**Pull request merged**